tests/tests.js

```diff
@@ -1087,20 +1084,14 @@ exports.defineManualTests = function (contentEl, createActionButton) {
     createActionButton('Seek To', function () {
         seekAudio('to');
     }, 'seekToBtn');
-    createActionButton('Set volume', function() {
+    createActionButton('Set volume', function () {
         setVolume();
     }, 'setVolumeBtn');
-    //get Special path to record if iOS || Blackberry
-    if (cordova.platformId === 'ios')
-        getRecordSrc();
-    else if (cordova.platformId === 'blackberry')
-        getRecordSrcBB();
-    else if (cordova.platformId === 'windows' || cordova.platformId === 'windows8')
-        getRecordSrcWin();
-
-    //testing process and details
-    function addItemToList(_list, _text)
-    {
+    // get Special path to record if iOS || Blackberry
+    if (cordova.platformId === 'ios') { getRecordSrc(); } else if (cordova.platformId === 'blackberry') { getRecordSrcBB(); } else if (cordova.platformId === 'windows' || cordova.platformId === 'windows8') { getRecordSrcWin(); }
```

Comment:
```suggestion
    if (cordova.platformId === 'ios') {
      getRecordSrc();
    } else if (cordova.platformId === 'blackberry') {
      getRecordSrcBB();
    } else if (cordova.platformId === 'windows' || cordova.platformId === 'windows8') { 
      getRecordSrcWin(); 
    }
```

Reply:
In this PR, I was only trying to convert to `eslint` with bare minimum changes, but thanks the suggestion.

Reply:
Which is exactly why I added the suggestion: Without that your commit changed the formatting of the if/else if block to one line instead of the correct multi line it was before.

Reply:
Alright, thanks for catching it!